src/lib/y2storage/proposal/autoinst_space_maker.rb

```diff
@@ -65,6 +65,7 @@ def cleaned_devicegraph(original_devicegraph, drives_map)
       # @option drive_spec [Boolean] "initialize" Initialize the device
       # @option drive_spec [String]  "use"        Partitions to remove ("all" or "linux")
       def delete_stuff(devicegraph, disk, drive_spec)
+        return unless disk.respond_to?(:partition_table)
```

Comment:
I guess this implies there is no way to specify in the profile what to do with the existing LVs in a VG. If that's the case, is not already wrong the fact that `#delete_stuff` is sometimes called on something that is not "partitionable"? (I assume that's the reason to for this `return`.